drones/image_processing/utils.py

```diff
@@ -0,0 +1,77 @@
+from typing import Tuple
+
+
+def calculate_focal(known_width: float, known_distance: float, pixel_width: int) -> float:
+    """Calculate focal length of the camera
+    focal_length = (pixel_width * known_distance) / known_width
+
+    Parameters:
+    ----------
+    known_width: float
+        measured width of the object (has to be known before running script)
+    known_distance: float
+        measured distance between detected object and camera (has to be known before running script)
+    pixel_width: int
+        width in pixels of detected object
+
+    Returns:
+    ----------
+    focal_length: float
+        calculated length of the focal
+    """
+    focal_length = (pixel_width * known_distance) / known_width
+
+    return focal_length
+
+
+def distance_to_camera(known_width: float, focal_length: float, pixel_width: int) -> float:
+    """Calculate distance between detected object and camera from equation
+    distance = (known_width * focal_length) / pixel_width
+
+    Parameters:
+    ----------
+    known_width: float
+        measured width of the object (has to be known before running script)
+    focal_length: float
+        length of the focal
+    pixel_width: int
+        width in pixels of detected object
+
+    Returns:
+    ----------
+    distance: float
+        Calculated distance between detected object and camera
+    """
+    distance = (known_width * focal_length) / pixel_width
+
+    return distance
+
+
+def vector_to_centre(frame_width: int, frame_height: int, obj_coordinates: Tuple, centre_height_coeff: float) -> Tuple:
```

Comment:
Please specify tuple members' type as well. Take a look [here](https://docs.python.org/3/library/typing.html#typing.Tuple) to see more examples.